pkg/options/options.go

```diff
@@ -30,7 +30,7 @@ const (
 	// ContainerImageRepo is the repo of the default image url
 	ContainerImageRepo = "noobaa-core"
 	// ContainerImageTag is the tag of the default image url
-	ContainerImageTag = "5.3.0-master20191219"
+	ContainerImageTag = "5.3.0-master20191223"
```

Comment:
Just making sure - is this a good image for anyone using master?

Reply:
suppose to fix some annoying issues: https://github.com/noobaa/noobaa-core/pull/5851